tools/testing-server/utils/forwarder.js

```diff
@@ -0,0 +1,39 @@
+const { gzip } = require('node-gzip')
+
+module.exports = async function forward (request, body) {
+  if (!request.url.startsWith('/debug') && !request.url.startsWith('/health')) {
+    const beacon = 'staging-bam-cell.nr-data.net'
+    const lk = 'NRBR-c7d7aa44f6c74d8ed93'
+    const a = 148790031
```

Comment:
We should probably make these flags to be passed into the cli along with a flag to enable the forwarder.